src/plugins/intel_gpu/src/kernel_selector/common/common_types.h

```diff
@@ -86,7 +86,8 @@ enum class KernelType {
     CONVERT_COLOR,
```

Comment:
if here is only format changes  could you revert it? better to do it in separate PR